compiler/rustc_lint/src/lints.rs

```diff
@@ -1434,17 +1434,15 @@ pub struct UnusedDef<'a, 'b> {
 }
 
 #[derive(Subdiagnostic)]
-pub enum UnusedDefSuggestion {
-    #[suggestion(
-        lint_suggestion,
-        style = "verbose",
-        code = "let _ = ",
-        applicability = "machine-applicable"
-    )]
-    Default {
-        #[primary_span]
-        span: Span,
-    },
+#[suggestion(
+    lint_suggestion,
+    style = "verbose",
+    code = "let _ = ",
+    applicability = "machine-applicable"
```

Comment:
This is not machine applicable, since it will often be wrong (imo it shouldn't be a structured suggestion like this at all). Often the fix is to deal with the value properly/use a different mutating function instead of just ignoring it.

Reply:
Obviously this was just copied from the other `let _ = ..` suggestion. I've updated both to `maybe-incorrect`.